src/scss/components/events-highlights.scss

```diff
@@ -0,0 +1,83 @@
+.event {
+  $root: &;
+  @extend .container;
```

Comment:
Instead of using the `@extend` of the class `container`, I think is clearer to add the class in the JS along with the class `event`.

Reply:
resolved